js/ui/camera.js

```diff
@@ -296,7 +296,9 @@ class Camera extends Evented {
      * Pans and zooms the map to contain its visible area within the specified geographical bounds.
      *
      * @memberof Map#
-     * @param {LngLatBoundsLike} bounds The bounds to fit the visible area into.
+     * @param {LngLatBoundsLike} bounds The bounds to fit the visible area into. If bounds contains
+     *     only one point (i.e. bounds.ne and bounds.sw are the same location, the map center
+     *     transitions to that point at zoom level options.maxZoom or {@link Map#getMaxZoom}.
```

Comment:
I understand now. `bounds.ne === bounds.sw` is an edge case that demonstrates what happens when the zoom that would best fit the bounds in the viewport is greater than `Map#getMaxZoom()`.

Is there a way we can phrase these docs to make the more general pattern evident?

Here's a rough sketch:

 > Center these bounds in the viewport and use the highest supported zoom level (up to and including `Map#getMaxZoom()`) that fits them in the viewport.